scripts/build_wheel.sh

```diff
@@ -21,5 +21,5 @@ make -f dlrover/Makefile
 
 # Create dlrover package
 echo "Building the wheel for dlrover."
-rm -rf ./build/lib
```

Comment:
Why do we need to remove the line?

Reply:
Just to reduce misunderstandings. Before this line, we have already executed `rm -rf build`. And the build file doesn't exist until we execute `python setup.py`.

Reply:
The command will not raise an error even if the `./build/lib` does not exist. So, it is better to retain it to clear the cache.